docs/machine-learning/tutorials/text-classification-tf.md

```diff
@@ -0,0 +1,260 @@
+  
+   Machine learning algorithms understand [featurized](../resources/glossary.md#feature) data, and when dealing with deep neural networks you must adapt the images to the format expected by the network. That format is a [numeric vector](../resources/glossary.md#numerical-feature-vector).
+
+   The [ResizeFeaturesAction](xref:System.Action%602) delegate resizes the integer vector to a fixed length vector for required model inputs. This will be used later by the CustomMapping transform.
+
+1. Use the [TokenizeIntoWords](xref:Microsoft.ML.TextCatalog.TokenizeIntoWords%2A) transform to break the text into words as the next line of code:
+
+   [!code-csharp[TokenizeIntoWords](~/samples/machine-learning/tutorials/TextClassificationTF/Program.cs#TokenizeIntoWords)]
+
+   The [TokenizeIntoWords](xref:Microsoft.ML.TextCatalog.TokenizeIntoWords%2A) transform uses spaces to parse the text/string into words. It creates a new column and splits each input string to a vector of substrings based on the user defined separator. Space is also a default value for the 'separators' argument if it is not specified.
+
+   [!code-csharp[MapValue](~/samples/machine-learning/tutorials/TextClassificationTF/Program.cs#MapValue)]
+
+   The [MapValue](Microsoft.ML.ConversionsExtensionsCatalog.MapValue%2A) transform maps each word to an integer which is an index in the dictionary `lookupMap` that you previously created.
+
+Add a `CustomMappingEstimator` to transform the data as the model expects the input feature vector to be a fixed length vector.
+
+[!code-csharp[CustomMapping](~/samples/machine-learning/tutorials/TextClassificationTF/Program.cs#CustomMapping)]
+
+The `ScoreTensorFlowModel` extracts specified outputs (the `sentiment_analysis model`'s sentiment prediction features (`Prediction`)), and scores a dataset using the pre-trained `TensorFlow` model:
+
+[!code-csharp[ScoreTensorFlowModel](~/samples/machine-learning/tutorials/TextClassificationTF/Program.cs#ScoreTensorFlowModel)]
+
+The [CopyColumns](xref:Microsoft.ML.TransformExtensionsCatalog.CopyColumns%2A) transform retrieves the `Prediction` from TensorFlow and copies to the `Features` column.
+
+[!code-csharp[CopyColumns](~/samples/machine-learning/tutorials/TextClassificationTF/Program.cs#CopyColumns)]
+
+The `Fit()` method trains your model by transforming the dataset and applying the training. Fit the model to the training dataset and return the trained model by adding the following as the next line of code in the `ReuseAndTuneSentimentModel()` method:
+
+[!code-csharp[TrainModel](~/samples/machine-learning/tutorials/TextClassificationTF/Program.cs#TrainModel)]
+
+Return the trained model by adding the following as the last line of code in the `ReuseAndTuneSentimentModel()` method:
+
+[!code-csharp[ReturnModel](~/samples/machine-learning/tutorials/TextClassificationTF/Program.cs#ReturnModel)]
```

Comment:
This whole pipeline section could do with a diagram. Just basic boxes would suffice